src/main/flight/pid.c

```diff
@@ -47,6 +47,27 @@
 #include "flight/imu.h"
 #include "flight/navigation_rewrite.h"
 
+/*
+    FIXME Here we have a depenency nighmare... try to resolve it by extending PID profile
+*/
+#include "io/gimbal.h"
+#include "sensors/barometer.h"
+#include "sensors/battery.h"
+#include "common/color.h"
+#include "io/ledstrip.h"
+#include "drivers/gpio.h"
+#include "drivers/serial.h"
+#include "drivers/timer.h"
+#include "io/serial.h"
+#include "telemetry/telemetry.h"
+#include "drivers/pwm_rx.h"
+#include "sensors/boardalignment.h"
+
+#include "config/config.h"
+#include "config/config_profile.h"
+#include "config/config_master.h"
+
+#define MAG_HOLD_ERROR_LPF_FREQ 2
```

Comment:
Yeap.... All because 2 masterConfig values are used in mag hold controller. 
I have an idea how to fix it, do in progress label stays in place